serveradmin/servershell/static/js/servershell/command.js

```diff
@@ -173,6 +173,23 @@ function validate_selected(min=1, max=-1) {
 }
 
 servershell.commands = {
+    pin: function() {
+        if (!Array.isArray(servershell.pinned)) {
+            servershell.pinned = [];
```

Comment:
Lets move the definition and initialisation [here](https://github.com/innogames/serveradmin/blob/main/serveradmin/servershell/templates/servershell/index.html#L214) to be consistent.